app/models/user_stat.rb

```diff
@@ -202,6 +203,38 @@ def update_distinct_badge_count
     self.class.update_distinct_badge_count(self.user_id)
   end
 
+  def self.update_draft_count(user_id = nil)
+    if user_id.present?
+      draft_count = DB.query_single <<~SQL, user_id: user_id
+        UPDATE user_stats
+        SET draft_count = (SELECT COUNT(*) FROM drafts WHERE user_id = :user_id)
+        WHERE user_id = :user_id
+        RETURNING draft_count
+      SQL
+
+      MessageBus.publish(
+        '/user',
+        { draft_count: draft_count.first },
+        user_ids: [user_id]
+      )
+    else
+      DB.exec <<~SQL
```

Comment:
I'm wondering if we need to reconcile periodically when we're confident that we're always updating the counter cache properly. Maybe just do away with this for now until a bug in the draft count shows up? Even if there is a spot we miss somewhere, creating or removing a new draft will fix the count.